lib/portlayer/network/context.go

```diff
@@ -858,6 +858,85 @@ func (c *Context) container(h *exec.Handle) (*Container, error) {
 	return nil, ResourceNotFoundError{error: fmt.Errorf("container %s not found", id.String())}
 }
 
+func (c *Context) UnbindContainerByID(id string) ([]*Endpoint, error) {
```

Comment:
This method has some duplications with `UnbindContainer`. Is it easy to consolidate these two methods?